README.md

```diff
@@ -18,7 +18,7 @@ As soon as we run a new build, your changes will automatically be included and a
 
 ## New Pages and Changes to the Structure
 
-New pages in our documentation will require some structural adjustments, which you need to discuss beforehand. If you have an idea on how to structure the documentation, please discuss your proposal on our [forum](https://forum.zettlr.com).
+New pages in our documentation will require some structural adjustments, which you need to discuss beforehand. If you have an idea on how to structure the documentation, please start a thread in the [discussions](https://github.com/Zettlr/Zettlr/discussions) tab of the main repository or join the [discord](https://discord.gg/StmkG4sR) server. 
```

Comment:
Please use the global invite link for Discord, makes it easier for search engines to recognize these are all pointing to the same server: https://discord.gg/PcfS3DM9Xj